admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -80,6 +80,17 @@ When you create a new WND share you need three things:
 - The server address, the share name; and
 - The folder you want to connect to
 
+the share. 
```

Comment:
This block is out of place. I think most of this content is already covered in the "wnd:listen" section, so we can remove this.